tdi_python/tdiTable.py

```diff
@@ -1802,11 +1804,40 @@ def set_supported_operations_to_supported_commands(self):
         keys_list = self.operations_type_cls.operations_dict.keys()
 
         # TODO: Enable this when operations are supported
-        '''
         for i in range(len(operations_arr)):
             if operations_arr[i] in keys_list:
                 self.supported_commands.append(self.operations_type_cls.operations_dict[operations_arr[i]])
-        '''
+
+    def set_supported_notifications_to_supported_commands(self):
+        num_notifs = c_uint(0)
+        sts = self._cintf.get_driver().tdi_table_num_notifications_supported(self._info_handle, byref(num_notifs))
+        if sts != 0:
+            raise TdiTableError("Error: num notifications supported get failed on table {}. [{}]".format(self.name, self._cintf.err_str(sts)), self, sts)
+
+        arr_type = c_char_p * num_notifs.value
+        notifs_arr = arr_type()
+        sts = self._cintf.get_driver().tdi_table_notifications_supported(self._info_handle, byref(notifs_arr), byref(num_notifs))
+
+        for notif in notifs_arr:
+            self.supported_commands += [notif.decode('ascii')] #self.attributes_type_cls.attributes_dict[notifs_arr[i]]
+
+        print("supported_commands: ", self.supported_commands)
```

Comment:
logging module and remove commented code

Reply:
Done

Reply:
I don't see this change reflected in the PR. Maybe it was missed?